platform/server-bundle/src/main/resources/webapp/WEB-INF/jsp/changePassword.jsp

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html ng-app="motech-dashboard">
+    <head>
+        <meta charset="UTF-8">
+        <title>MOTECH - Mobile Technology for Community Health</title>
+
+        ${mainHeader}
+
+    </head>
+
+    <body ng-show="ready" ng-controller="MotechMasterCtrl" class="body-down" ng-init="initChangePasswordViewData()">
```

Comment:
One problem with this page I noticed is that the button only becomes active after we insert the correct data in the confirm password input. If for example I first enter the new password, then enter a confirmation password that does not match, then go back to edit the password in the first input - then the validation errors will dissapear but the button will remain greyed out until I do something in the confirm password input.